clientdb/order.go

```diff
@@ -6,11 +6,23 @@ import (
 	"fmt"
 	"io"
 
+	"github.com/btcsuite/btcutil"
 	"github.com/lightninglabs/pool/event"
 	"github.com/lightninglabs/pool/order"
+	"github.com/lightningnetwork/lnd/tlv"
 	"go.etcd.io/bbolt"
 )
 
+const (
+	// selfChanBalanceType is the tlv type we use to store the self channel
+	// balance on bid orders.
+	//
+	// NOTE: When adding new tlv types for extra order data, make sure the
+	// types don't overlap between bid and ask orders! Always increase the
+	// type by one for each new one.
+	selfChanBalanceType tlv.Type = 1
```

Comment:
Re this comment, can we prefix the _name_ of the type with `ask` or `bid`, then be able to safely have both type streams share the same values if we need to?